topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -39,7 +42,11 @@ def is_applicable(self, asset: Asset) -> bool:
 
     def load_metadata(self, asset: Asset) -> None:
         if not self.is_init:
-            self.read_csv()
+            if not is_s3_path(asset.source_path):
```

Comment:
what is the use case for using `asset.source_path` here? It should always just go to the `lds_cache`

Reply:
Unit tests, should find another way but I was not inspired

Reply:
could make `read_csv` take the file path then call the `.read_csv(asset_path)` in the unit test setup before processing files

Reply:
Will go along with https://github.com/linz/topo-processor/pull/457